feat: deprecate reactor formulas and remove references to it (#249)
Basis-Theory/developers.basistheory.com: docs/api/testing.mdx

## Testing Reactors

[Reactor code](/docs/api/reactors#reactor-code) is simply an ES module that exports a default function.
This can be tested like any JavaScript code or ES module using the tooling of your choice.

> Internally at Basis Theory, we unit and integration test our reactor code using Jest to ensure the JavaScript code itself works as expected.

Reviewer: ```suggestion
Internally at Basis Theory, we have automated tests for our reactor code using Jest to ensure the JavaScript code itself works as expected.
```
